content/rs/security/ldap/migrate-to-role-based-ldap.md

```diff
@@ -43,9 +39,9 @@ This checklist covers the basic process:
 
  Because deployments and requirements vary, you’ll likely need to adjust these guidelines.
 
-## Ways to test access
+## Test LDAP access
 
-There are several ways to test your LDAP integration, including:
+To test your LDAP integration, including:
 
 - Connecting with `redis-cli` and using the AUTH command to test LDAP username/password credentials.
```

Comment:
Maybe remove the -ing verbs in this list of test methods.
```suggestion
- Connect with `redis-cli` and use the `AUTH` command to test LDAP username/password credentials.
```

Reply:
Fixed.